numpy/distutils/ccompiler_opt.py

```diff
@@ -365,7 +371,8 @@ class attribute `conf_features`, also its override
             AVX512_CNL = dict(flags="-mavx512ifma -mavx512vbmi"),
             AVX512_ICL = dict(
                 flags="-mavx512vbmi2 -mavx512bitalg -mavx512vpopcntdq"
-            )
+            ),
+            AVX512_SPR = dict(flags="-mavx512fp16")
```

Comment:
If you use trailing commas in this file, the next addition won't have to touch the previous line.